tests/integration/test_charm_nrpe.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Tests for jenkins relation with nrpe."""
+
+import pytest
+import pytest_asyncio
+from pytest_operator.plugin import OpsTest
+from ops.model import Application, ActiveStatus
+
+pytestmark = pytest.mark.nrpe
+
+
+NRPE_CHECK_HTTP_FILE = "/usr/lib/nagios/plugins/check_http"
+NRPE_CONFIG_FILE = "/etc/nagios/nrpe.d/check_check_jenkins_http.cfg"
+NAGIOS_CMD = "sudo $(grep -e '{}' {} | cut -d'=' -f2-)".format(
+    NRPE_CHECK_HTTP_FILE, NRPE_CONFIG_FILE
+)
+NAGIOS_CMD_SUCCESS = "HTTP OK"
+
+
+@pytest_asyncio.fixture(scope="module")
+async def nrpe(app_name: str, ops_test: OpsTest, app: Application, series: str):
+    """Add relationship with nrpe and nagios to app."""
+    nrpe_app: Application = await ops_test.model.deploy("nrpe", series=series)
+    await ops_test.model.add_relation(
+        "{}:nrpe-external-master".format(app_name), "nrpe:nrpe-external-master"
+    )
+    # [2022-09-29] Nagios does not support focal
```

Comment:
Can you file a bug and reference the bug here? (Although see my comment below - I don't think you actually need nagios here).

Reply:
It seems that focal won't be supported: https://bugs.launchpad.net/charm-nagios/+bug/1900698